src/components/Route/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as ProtectedRouteWithLayout } from './ProtectedRouteWithLayout';
+export { default as RouteWithLayout } from './RouteWithLayout.';
```

Comment:
Remove the final dot in './RouteWithLayout.' because the file does not compile

Reply:
Fixed